mobile/src/containers/onboarding/Files.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import OnBoarding from '../../components/OnBoarding'
+
+export const Files = ({ onActivate }) =>
+(
+  <OnBoarding onActivate={onActivate} nameStep='files' currentStep={1} totalSteps={3} />
```

Comment:
I think it'll be hard to maintain that `currentStep` magic number.
The authority (the entity that knows what are the current step and the steps total) must be the parent component.